misc/workflow-environment-basic-dependencies-without-workspace/README.md

```diff
@@ -0,0 +1,33 @@
+## What is this?
+
+This is an example of a workflow template. In here, you define all environments and deployments that take part in the workflow.
+This allows you to create stack dependencies.
+
+In this specific scenario: 
+- Initial deployment would start deployment of two stacks: `rootService1` and `rootService2`
+- Once both "root" services finish deploying, two other stacks start deployment - `middleService1` and `middleService2`
+- Once `middleService1` finishes, it triggers `finalService` stack deployment
+- Once all above stacks have fully deployed, the deployment is done
+
+This example deploys `null resource` which means this is purely for example purposes and does not actually deploy anything. 
+
+## How to run in env0?
+
+### Prerequisites
+
+This example assumes you have a template named `null` in your organization.
+
+To successfully run - you can either create that template in your organization or alter `env0.workflow.yml` to match a different template name.
+
+### Defining the template
+
+- In env0, create a new template of type "env0 workflow"
+- On the VCS step, pick the repository containing this file, and put `misc/workflow-environment` under "env0 Workflow Location"
```

Comment:
`misc/workflow-environment` <- isn't this wrong?

Reply:
bdca8a0180eb6d619665e5cd807e0b34f992d4b1